generate-sdks.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+import * as colors from "colors";
+import * as fs from "fs";
+import * as path from "path";
+import * as minimist from "minimist";
+
+const repositoryName = "azure-rest-api-specs";
+const specificationsSegment = "specification";
+
+const contains = function (array, el) {
```

Comment:
Can you make this a normal function instead of a variable pointing to a function?